### ISSUE TITLE
Tax-Calculator input output data frame warning

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/Tax-Calculator#2626]
Users of Tax-Calculator have noticed a warning like the following:

```
C:\Users\rh2845\Anaconda3\lib\site-packages\taxcalc\taxcalcio.py:702: PerformanceWarning: DataFrame is highly fragmented. This is usually the result of calling frame.insert many times, which has poor performance. Consider joining all columns at once using pd.concat(axis=1) instead. To get a de-fragmented frame, use newframe = frame.copy()
odf[varname] = vardata.round(2) # rounded to nearest cent
```

As the warning suggests, it looks like new variables may be added to the `odf` DataFrame in a more efficient way.

**Comment by jackisace** [comment]
I don't have permissions to contribute, but my fix would be to add :
`odf = odf.copy()`
at the end of the loop. This will defragment the dataframe on each pass.

This will result in a slower runtime but will maintain data fidelity.

**Comment by BashCache** [comment]
Hi, can I work on this

**Comment by jdebacker** [comment]
@BashCache Thanks for the offer, but this was just resolved last night.

**Comment by jdebacker** [comment]
Resolved with PR #2627
